component/core/src/main/java/org/exoplatform/social/core/listeners/ActivityTagMetadataListener.java

```diff
@@ -79,12 +75,7 @@ private void updateActivityTags(ExoSocialActivity activity) {
 
     Identity audienceIdentity = activityManager.getActivityStreamOwnerIdentity(activity.getId());
     long audienceId = Long.parseLong(audienceIdentity.getId());
-    String content = getActivityBody(activity);
-    try {
-      content = HTMLSanitizer.sanitize(content);
-    } catch (Exception e) {
-      LOG.warn("Error while sanitizing activity content {}", content, e);
-    }
+    String content = StringEscapeUtils.unescapeHtml(getActivityBody(activity));
```

Comment:
we may need to unescape HTML in `tagService.detectTagNames` instead to centralize this behavior + a UT please

Reply:
done